common/build.gradle

```diff
@@ -208,7 +208,7 @@ dependencies {
     testFixturesImplementation group: 'com.google.guava', name: 'guava', version: '29.0-jre'
     testFixturesImplementation group: 'de.flapdoodle.embed', name: 'de.flapdoodle.embed.mongo', version: '3.3.1'
     testFixturesImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
-    testFixturesImplementation group: 'org.mongodb', name: 'mongodb-driver-sync', version: '4.4.1'
+    testFixturesImplementation group: 'org.mongodb', name: 'mongodb-driver-sync', version: '4.5.0'
```

Comment:
Do we need this line if we already have it in the `implementation` section?

Reply:
@andiem-bq Do we need this line if we already have it in the `implementation` section?